addons/stock/models/stock_move.py

```diff
@@ -598,6 +598,13 @@ def write(self, vals):
                 receipt_moves_to_reassign |= move_to_unreserve.filtered(lambda m: m.location_id.usage == 'supplier')
                 receipt_moves_to_reassign |= (self - move_to_unreserve).filtered(lambda m: m.location_id.usage == 'supplier' and m.state in ('partially_available', 'assigned'))
                 move_to_recompute_state |= self - move_to_unreserve - receipt_moves_to_reassign
+        # propagate packaging_id changes in the stock move chain
+        if 'product_packaging_id' in vals:
+            if self.move_dest_ids and \
+                    self.move_dest_ids.state not in ['cancel', 'done'] and \
+                    self.move_dest_ids.product_packaging_id.id != vals['product_packaging_id'] and \
```

Comment:
Both lines will raise an error if there are several move_dest_ids (and possibly several product_packaging_id)
Can you add a test that protects this error ?